.ruby-version

```diff
@@ -0,0 +1 @@
+2.2.2
```

Comment:
Ruby version files in gems don't affect the applications they're in, they're just used as an environment for running tests and building the gem.  See https://github.com/alphagov/gds-api-adapters/blob/master/.ruby-version